src/app/main/component/shared/components/input-error/input-error.component.ts

```diff
@@ -38,20 +43,32 @@ export class InputErrorComponent implements OnInit {
             this.errorMessage = this.getMinlengthErrorMessage(this.formElement.errors.minlength.requiredLength);
             break;
           case errorType.maxlength:
-            this.errorMessage = this.getMaxlengthErrorMessage(this.formElement.errors.maxlength.requiredLength);
+            this.errorMessage = this.getMaxlengthErrorMessage(this.formElement.errors.maxlength.requiredLength, this.isEvent);
             break;
           default:
+            if (this.isEvent) {
```

Comment:
don't use if here
this.errorMessage = this.isEvent ? this.getRequiredErrorMessage() : this.validationErrors[err]